### ISSUE TITLE
Quest By Blood And ash

### ISSUE THREAD

**Issue by LKArthas** [SkyFireArchives/SkyFireDB#169]
this quest its working almost fine, the problems taht have is that in the quest description gives a maximun of kills of 80 and you complete it with just 40, the other bug in this quest is that you can move the rebels cannons freely

**Comment by Nomadgraphics** [comment]
Yes we know it has issues, if you haven't guessed yet... we are going through the whole quest line fixing them one by one. Hang in there man we will have it soon.

**Comment by LKArthas** [comment]
Thanks you nomad, i just surprised because that quest was working well
in previous versions of the DB, don´t worry i will wait ;-)
